app/controllers/posts_controller.rb

```diff
@@ -4,6 +4,21 @@ def index
     @user = User.find(params[:user_id])
   end
 
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = current_user.posts.new(params.require(:post).permit(:title, :text))
```

Comment:
- [ OPTIONAL ] It would be better if you create a private post_params method. You can visit this lesson link about the strong parameters: [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides).